src/java.base/share/classes/java/security/KDFParameters.java

```diff
@@ -38,8 +38,12 @@
  * {@link javax.crypto.KDF#getInstance(String, KDFParameters) KDF.getInstance}
  * methods. The {@code getInstance} method returns a {@code KDF}. The
  * {@code KDF} that is returned is initialized with the specified parameters.
```

Comment:
Suggest combining the above 2 sentences to:

The {@code getInstance} method returns a {@code KDF} that is initialized with the specified parameters.